SM-6004: add docs for SM-6004 - work extension CR fix (#826)
departmentfortransport/street-manager-docs: docs/api-documentation/versions-and-changes/v2/changelog.md

{: .govuk-body}
<ol class="govuk-list govuk-list--bullet">
  <li>SM-7060: <code>PermitResponse</code> and <code>PermitAlterationResponse</code> have been updated to return <code>duration_challenge_non_acceptance_response_old_reasonable_period_end_date</code>.
  </li>

>   <li>SM-6004: Fixed a bug with work extensions on works in progress to not flag an early start if the work did not originally have an early start.</li>

Reviewer: Can we call out the endpoint the this affects and state the difference. i.e. updated <endpoint> to flag is_early_start false when works in progress etc. etc.

Author: @cpaul92  
This actually affects a few:
GET permits/category
POST works/{workReferenceNumber}/permits/{permitReferenceNumber}/alterations
POST works/

Because they all access the WorkCategoryService.getWorkCategory.
So I can add this here

>   <li>SM-6004: Fixed a bug with work extensions on works in progress that are not flagged with an early start. The frontend no longer shows the early start screens.</li>

Reviewer 2: Wonder if something like the following makes more sense for API users, i.e. to describe the bugfix without referencing the frontend?
```suggestion
  <li>SM-6004: Fixed a bug with work extensions on works in progress to not flag an early start if the work did not originally have an early start.</li>
```
